addons/website_project/models/project_task.py

```diff
@@ -8,3 +8,7 @@ class ProjectTask(models.Model):
 
     # Need this field to check there is no email loops when Odoo reply automatically
     email_from = fields.Char('Email From')
+    # Used to submit tasks from a contact form
+    partner_name = fields.Char(string='Customer Name', related="partner_id.name", readonly=False)
+    partner_phone = fields.Char(string='Customer Phone', related="partner_id.phone", readonly=False)
+    partner_company_name = fields.Char(string='Company Name', related="partner_id.company_name", readonly=False)
```

Comment:
Are you sure we don't need to store them? if yes then okay let's merge that task. 🙂

Reply:
yes, No need to store them. we can merge it.